tmol/score/ljlk/potentials/lk_isotropic.hh

```diff
@@ -128,11 +128,22 @@ struct lk_isotropic_pair {
 
     Real lk;
 
-    if (dist < cpoly_close_dmin) {
-      lk = f_desolv<Real>::V(
-          d_min, lj_radius_i, lk_dgfree_i, lk_lambda_i, lk_volume_j);
+    if (dist > cpoly_far_dmax) {
```

Comment:
ooo, is it faster to exit early from this function?

Reply:
So I tested this, thinking it might be ... unfortunately, it did not make a difference.